React/Views/RCTViewManager.m

```diff
@@ -120,6 +120,7 @@ - (RCTViewManagerUIBlock)uiBlockToAmendWithShadowViewRegistry:(__unused NSDictio
 RCT_REMAP_VIEW_PROPERTY(shadowOpacity, layer.shadowOpacity, float)
 RCT_REMAP_VIEW_PROPERTY(shadowRadius, layer.shadowRadius, CGFloat)
 RCT_REMAP_VIEW_PROPERTY(overflow, clipsToBounds, css_clip_t)
+RCT_REMAP_VIEW_PROPERTY(zIndex, layer.zPosition, CGFloat)
```

Comment:
There are a couple of problems with using layer.zPosition for this:

1) It doesn't affect event processing order, so views that are visually in behind another view because of zIndex, but appear later in the hierarchy order, won't receive touch events because the other view will intercept them,.

2) layer.zPosition doesn't just affect order, it actually moves the view in 3D. If you are using a perspective transform matrix that will affect the scale and position of the view due to parallax, as it will be closer or further from the camera.